content/en/scale-agent/concepts/dynamic-account-options.md

```diff
@@ -0,0 +1,56 @@
+
+For each account three requests are required to activate a migrated account on the Agent and set that account to `Active`:
+1. First use a `GET /armory/accounts/{accountId}` request to scan for account data. You can also use the `GET /armory/accounts/{accountId}` endpoint to source a specific account.
+2. Next, use the migrate `POST /armory/accounts` request to migrate the accounts to the Agent. This can be done as a batch process. Identified accounts are sent to the agent and set to an `INACTIVE` state.
+3. Set the migrated account to `ACTIVE` using a `PATCH /armory/accounts` request, after the migration step is completed.
+
+The initial request from the Agent to Clouddriver sets up the persistent connection, including the list of static accounts, defined in the agent configuration file `armory-agent.yml`, and by the optional `zoneId` flag. Where the flag is not applied migration still occurs. In this case accounts are polled at regular intervals and assigned to Agent pods. 
+
+Use of the `zoneId` flag is recommend for efficient migration. The `kubeconfig` file provides configuration details for connecting to the target clusters that are sent as part in a request. 
+
+If the agent is not able to connect to the target clusters, the Clouddriver sends the `SaveAccounts` operation to an Agent with a unique zone identifier. On demand HTTP requests indicate what accounts are removed from Agents and managed by Clouddriver.
+
+#### Interceptor option
+An interceptor option is provided to ease the configuration steps. With this option the configuration steps are minimized and the new account is enabled with only one request to set it to `ACTIVE` from the initial `INACTIVE` state.
+
+>This endpoint does not allow batch requests.
+
+### Autoscan
+The Autoscan configuration polls the connected database (SQL is supported for beta) for account activity, at regular intervals. The default is 2 minutes, and can be modified in the service plugin.  New accounts are identified with a customer defined prefix to the account name and requests are sent to the Clouddriver to migrate the account data to the Agent. In this case the type field is set to `kubernetes:agent` to indicate that the account is migrated to the Agent. When the type field is changed from `kubernetes` to `kubernetes:agent`, any field under `kubernetes.accounts`, in [Agent advanced config options](https://docs.armory.io/armory-enterprise/armory-agent/advanced-config/agent-options/), is monitored.
+
+The initial request from the Agent to Clouddriver sets up the persistent connection, including the list of static accounts defined in the Agent `armory-agent.yml` configuration file. The Clouddriver notifies the Agent about accounts that are configured for Agent monitoring and management, and provides any new, updated or deleted account data. The `kubeconfig` file targets and cluster connection configuration is also sent.  
```

Comment:
LEt's remove this paragraph as it explains general agent behavior not strictly related to dynamic accounts.